File: JournalLabs.API/ViewModels/StudentLabBlocksViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using JournalLabs.API.Models;

namespace JournalLabs.API.ViewModels
{
    public class StudentLabBlocksViewModel
    {
        public Student StudentInfo { get; set; }
        public List<LabBlockViewModel> StudentLabBlocks { get; set; }
    }
}

File: ClientApp/app/models/studentLabBlocksViewModel.ts
import { Student } from "./Student"
import { LabBlock } from "./LabBlock"
import { Remark } from "./Remark"

export class StudentLabBlocksViewModel {
  StudentInfo: Student = new Student();
  StudentLabBlocks: LabBlock[] = [];
  RemarkText: Remark = new Remark();
}
